### DISCUSSION TITLE
Error while installing code server on raspberry 4

### DISCUSSION THREAD

**Discussion by salvuccicarlos** [coder/code-server#3806] [General] [answered]
I'm having this error while trying to install code server on raspberry 4 with an armv7l processor. I have node v14.17.3 and npm 7.20.0 correctly installed

Raspbian GNU/Linux 10 (buster)
No standalone releases for armv7l.
Falling back to installation from npm.
Installing latest from npm.

Please install npm or yarn to install code-server!
You will need at least node v12 and a few C dependencies.
See the docs https://coder.com/docs/code-server/v3.11.0/install#yarn-npm

**Comment by jsjoeio** [comment, marked as answer]
> Please install npm or yarn to install code-server!
> You will need at least node v12 and a few C dependencies.
> See the docs https://coder.com/docs/code-server/v3.11.0/install#yarn-npm

And have you tried following that? What happens if you install with `npm` or `yarn`?

**Reply by jvadair** [reply]
(rpi 4b) I tried installing with npm and received several errors, but installing with yarn worked fine for me (so I would recommend using yarn).

**Reply by jsjoeio** [reply]
Awesome! Good to know. Yeah there might be some weird quirks with `npm`. Feel free to submit a PR to update the docs! Otherwise we'll eventually get around to it
